pegasus/sites.v3/code.org/views/inspirational_videos.haml

```diff
@@ -16,4 +16,4 @@
 %div{style: 'clear: both'}
 %br/
 -if request.locale == 'en-US'
-  %p= I18n.t(:inspirational_videos_more)
+  %p{style: 'padding: 10px'}= I18n.t(:inspirational_videos_more)
```

Comment:
Well maybe in a general css file it would already be included... I'll hunt for a good spot.